Logger.Tests/LogFactoryTests.cs

```diff
@@ -6,6 +6,49 @@ namespace Logger.Tests
     [TestClass]
     public class LogFactoryTests
     {
-        
+        [TestMethod]
+        public void CreateLogger_FileLoggerNotConfigured_ReturnNull()
+        {
+            LogFactory? lf = new LogFactory();
```

Comment:
```suggestion
            LogFactory? logFactory = new LogFactory();
```
I read this as `if` initially. :)